xla/translate/mhlo_to_lhlo_with_xla/mhlo_to_lhlo_with_xla.cc

```diff
@@ -1831,10 +1831,7 @@ tsl::StatusOr<mlir::Operation*> LhloDialectEmitter::EmitAsyncStartOp(
     case xla::HloOpcode::kAllToAll:
       return EmitAllToAllStartOp(instr);
     default:
-      return tsl::errors::InvalidArgument(
-          "Unexpected instruction %s wrapped in %s",
-          xla::HloOpcodeString(async->async_wrapped_opcode()),
-          HloOpcodeString(instr->opcode()));
+      return EmitEmptyAsyncStartOp(instr);
```

Comment:
I think we should modify EmitAsyncStartOp to emit the dummy op, instead of emitting an it as the default case.